client/my-sites/promote-post/index.js

```diff
@@ -42,4 +43,14 @@ export default () => {
 		makeLayout,
 		clientRender
 	);
+
+	page(
```

Comment:
We will also need this in the i2 controller. The Blaze app uses that controller, and in the future, we are planning to remove this one.

Reply:
I saw that the /promote-post-i2/ page definitions seemed to me that we weren't using them. Not 100% sure, but otherwise the page wouldn't be opening.

But yes, you're right, we need to keep a copy of the i1 into i2 folder